scripts/main/lychee.js

```diff
@@ -731,33 +863,45 @@ lychee.setMode = function (mode) {
 	header.bind_back();
 };
 
+/**
+ * @param {jQuery} obj
+ * @param {string} animation
+ *
+ * @returns {void}
+ */
 lychee.animate = function (obj, animation) {
-	let animations = [
+	const animations = [
 		["fadeIn", "fadeOut"],
 		["contentZoomIn", "contentZoomOut"],
 	];
 
-	if (!obj.jQuery) obj = $(obj);
-
 	for (let i = 0; i < animations.length; i++) {
 		for (let x = 0; x < animations[i].length; x++) {
-			if (animations[i][x] == animation) {
+			if (animations[i][x] === animation) {
 				obj.removeClass(animations[i][0] + " " + animations[i][1]).addClass(animation);
-				return true;
 			}
 		}
 	}
-
-	return false;
 };
 
+/**
+ * DON'T USE THIS METHOD.
+ *
+ * TODO: Find all invocations of this method and nuke them.
```

Comment:
I don't believe we use it anymore. I would delete it and see if anything breaks...

Reply:
Your are right, my IDE is actually telling me that this is dead code and hence code smell. I will nuke it. But wonderful that I annotated it in the first place.